recipes-extended/ostree/ostree_%.bbappend

```diff
@@ -8,6 +8,7 @@ SRC_URI:append = " \
     file://0003-ostree-fetcher-curl-set-max-parallel-connections.patch \
     file://0001-mount-Allow-building-when-macro-MOUNT_ATTR_IDMAP-is-.patch \
```

Comment:
nitpick: Shouldn't the new patch be listed here (right after the `0003-ostree-fetcher-...`)?

Reply:
Btw, I've just noticed these two patches are already upstream (since v1.0.4 of composefs-tools (now composefs)):

- 0001-mount-Allow-building-when-macro-MOUNT_ATTR_IDMAP-is-.patch
- 0002-mount-Allow-building-when-macro-LOOP_CONFIGURE-is-no.patch

Notice the patches are within the `ostree` recipe but actually apply to its `composefs` submodule.

Reply:
Nice, thanks @rborn-tx!

I've added a new commit here removing these two composefs patches, so now the ostree commit has the patches correctly ordered.